pallets/staking/src/lib.rs

```diff
@@ -1233,36 +1377,36 @@ decl_event!(
     pub enum Event<T> where Balance = BalanceOf<T>, <T as frame_system::Trait>::AccountId {
         /// The era payout has been set; the first balance is the validator-payout; the second is
         /// the remainder from the maximum amount of reward.
+        /// [era_index, validator_payout, remainder]
         EraPayout(EraIndex, Balance, Balance),
-        /// The staker has been rewarded by this amount. `AccountId` is the stash account.
+        /// The staker has been rewarded by this amount. [stash, amount]
         Reward(AccountId, Balance),
         /// One validator (and its nominators) has been slashed by the given amount.
+        /// [validator, amount]
         Slash(AccountId, Balance),
         /// An old slashing report from a prior era was discarded because it could
-        /// not be processed.
+        /// not be processed. [session_index]
         OldSlashingReportDiscarded(SessionIndex),
-        /// A new set of stakers was elected with the given computation method.
+        /// A new set of stakers was elected with the given [compute].
         StakingElection(ElectionCompute),
-        /// A new solution for the upcoming election has been stored.
+        /// A new solution for the upcoming election has been stored. [compute]
         SolutionStored(ElectionCompute),
-        /// An account has bonded this amount.
+        /// An account has bonded this amount. [stash, amount]
         ///
         /// NOTE: This event is only emitted when funds are bonded via a dispatchable. Notably,
         /// it will not be emitted for staking rewards when they are added to stake.
-        Bonded(IdentityId, AccountId, Balance),
-        /// User has unbonded their funds
-        Unbonded(IdentityId, AccountId, Balance),
+        Bonded(AccountId, Balance),
+        /// An account has unbonded this amount. [stash, amount]
+        Unbonded(AccountId, Balance),
         /// User has updated their nominations
-        Nominated(IdentityId, AccountId, Vec<AccountId>),
```

Comment:
AFAIR this is Polymesh related change as we do want to emit and IdentityId here in all the events.